packages/functions/src/coordinator/GetCoordinatorAppointmentsHandler.ts

```diff
@@ -71,22 +71,33 @@ async function fetchCoordinator(callerId: string) {
   return coordinator;
 }
 
-async function validate(coordinator: DbCoordinator, hospital: Hospital) {
+async function validate(
```

Comment:
Name is no longer right, as this not only validates.

How about `getValidHospitalsOrThrow`?

Reply:
done